docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -596,15 +596,16 @@ quarkus.oidc-client.token-path=/protocol/openid-connect/tokens
 [[token-propagation]]
 == Token Propagation in MicroProfile RestClient client filter
 
-`quarkus-oidc-token-propagation` extension provide `io.quarkus.oidc.token.propagation.AccessTokenRequestFilter` and `io.quarkus.oidc.token.propagation.JsonWebTokenRequestFilter` JAX-RS ClientRequestFilters which propagates the current link:security-openid-connect[Bearer] or link:security-openid-connect-web-authentication[Authorization Code Flow] access token as an HTTP `Authorization` `Bearer` scheme value.
+The `quarkus-oidc-token-propagation` extension provides two JAX-RS `javax.ws.rs.client.ClientRequestFilter` class implementations that simplify the propagation of authentication information.
+`io.quarkus.oidc.token.propagation.AccessTokenRequestFilter` propagates the current link:security-openid-connect[Bearer], while `io.quarkus.oidc.token.propagation.JsonWebTokenRequestFilter` propagates the link:security-openid-connect-web-authentication[Authorization Code Flow] access token as an HTTP `Authorization` `Bearer` scheme value.
```

Comment:
@geoand Hi Georgios, no, it is not correct:
```
io.quarkus.oidc.token.propagation.AccessTokenRequestFilter` propagates the current link:security-openid-connect[Bearer], while `io.quarkus.oidc.token.propagation.JsonWebTokenRequestFilter` propagates the link:security-openid-connect-web-authentication[Authorization Code Flow] access token as an HTTP `Authorization` `Bearer` scheme value.
``` 

The access token can be a bearer token coming with HTTP Authorization or acquired with the code flow - both filters work for both cases - the difference is, the JWT one adds some extra support for tokens which are in JWT format, so I'd drop this note

Reply:
Aha, thanks.

I'll update the docs then.

Reply:
Hopefully it's better now

Reply:
@geoand Yes, thanks